bowling.test.js

```diff
@@ -0,0 +1,101 @@
+const Bowling = require('./bowling')
+
+describe("Bowling", () => {
+    it('can return the score of 0 before any rolls', () => {
+        const game = new Bowling;
```

Comment:
Could use a before clause to avoid repetition around starting a new game instance.